cal-itp/benefits
# Feat: CalFresh enrollment index

Closes #1917 

~Currently on top of #1970 since both PRs introduce new migrations.~

~Currently on top of #1979 since both PRs introduce new migrations.~

## Todo

- [x] Fix spacing between media item and alert box (should be `48px`, currently the default of `64px`)
- [x] Fix horizontal alignment of the alert box so it is aligned with the media item

## Testing

* Run `bin/init.sh` to run migrations
* Launch the app with `F5`
* In the Admin, update the `enrollment_index_template` for a flow where you can pass eligibility, e.g. `MST Courtesy Cards`, to be `enrollment/index--calfresh.html`
* _For the full experience, update the corresponding `PaymentProcessor` instance to connect to the QA environment_
* Pass eligibility for the flow
* See the new CalFresh enrollment index with alert box:

![image](https://github.com/cal-itp/benefits/assets/1783439/576bd9ed-fee2-4252-9509-9a8033f0b7bc)
